packages/cloud-cognitive/package.json

```diff
@@ -37,13 +37,15 @@
     "build:scss": "bundler bundle:scss src/index.scss && copyfiles 'src/**/*.scss' scss -u 1",
     "ci-check": "node scripts/import",
     "clean": "rimraf es lib scss",
+    "postinstall": "carbon-telemetry collect --install",
     "test": "bundler check 'src/**/*.scss'"
   },
   "peerDependencies": {
     "react": "^16.13.1"
   },
   "dependencies": {
     "@carbon/ibm-cloud-cognitive-experimental": "^0.26.8",
-    "@carbon/ibm-cloud-cognitive-security": "^0.4.11"
+    "@carbon/ibm-cloud-cognitive-security": "^0.4.11",
+    "@carbon/telemetry": "^0.0.0-alpha.6"
```

Comment:
This adds a new dependency. Have we done the review checks?

Reply:
Good point.
1. It's Carbon not sure this is applicable.
2. What is the process for this? https://www.npmjs.com/package/@carbon/telemetry?activeTab=dependencies
3. 27.3 kB unpacked, but not added to our component dist.
4. Spoke with Josh earlier and it is still alive.
5. License is the standard Carbon Apache-2.0 License.

Reply:
Agree on (1).

For (2), is `yarn audit` sufficient? If so, I can confirm that `yarn audit` does not report any vulnerabilities for @carbon/telemetry.

Reply:
Happy to go with 'yarn audit' as a step. We should probably have an explainer for these checkboxes in a doc somwhere.